### ISSUE TITLE
[bug]: generate entity panic

### ISSUE THREAD

**Issue by jacksoom** [SeaQL/sea-orm#445]
```sh
sea-orm-cli generate entity -o src/entity
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Database(MySqlDatabaseError { code: Some("42S22"), number: 1054, message: "Unknown column 'generation_expression' in 'field list'" })', /Users/jacksoom/.cargo/registry/src/github.com-1ecc6299db9ec823/sea-schema-0.2.9/src/mysql/discovery/executor/real.rs:32:14

**Comment by billy1624** [comment]
Hey @jacksoom, can I know the MySQL database version that you are using? I think this issue is specific to a database version.

**Comment by jacksoom** [comment]
mysql version: 5.5.62-log

verbose backtrace
[2022-01-13T02:35:37Z INFO  sqlx::query] SELECT `column_name`, `column_type`, `is_nullable`, …; rows: 0, elapsed: 31.262ms

    SELECT
      `column_name`,
      `column_type`,
      `is_nullable`,
      `column_key`,
      `column_default`,
      `extra`,
      `generation_expression`,
      `column_comment`
    FROM
      `information_schema`.`columns`
    WHERE
      `table_schema` = ?
      AND `table_name` = ?
    ORDER BY
      `ordinal_position` ASC

thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Database(MySqlDatabaseError { code: Some("42S22"), number: 1054, message: "Unknown column 'generation_expression' in 'field list'" })', /Users/jacksoom/.cargo/registry/src/github.com-1ecc6299db9ec823/sea-schema-0.2.9/src/mysql/discovery/executor/real.rs:32:14
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Is it related to the mysql engine?  because there are tables with different engines （myisam and innodb）

**Comment by billy1624** [comment]
Thanks!

**Comment by billy1624** [comment]
So the column `GENERATION_EXPRESSION ` was added since `5.7` any MySQL version below that won't have the column.

The INFORMATION_SCHEMA COLUMNS Table:
- MySQL 5.6: https://dev.mysql.com/doc/refman/5.6/en/information-schema-columns-table.html
- MySQL 5.7: https://dev.mysql.com/doc/refman/5.7/en/information-schema-columns-table.html

**Comment by billy1624** [comment]
> > Thanks!
> 
> there is no generation_expression field in my database table `information_schema`.`columns`. should i update the mysql version . can you give me some advive for this

I'm not sure is there any breaking changes when upgrading from `5.5.62-log` to `5.7.x`. But I'll check why we need `GENERATION_EXPRESSION` in a minute.

**Comment by billy1624** [comment]
Hey @tyt2y3, I want to know why we need the `expression: Option<ColumnExpression>`? Seems that it's not being used in the writer.

https://github.com/SeaQL/sea-schema/blob/bd742757bd2dc609c822daccd3ab426144a9804b/src/mysql/def/column.rs#L6-L25

**Comment by heliping** [comment]
same problem. any update?
![image](https://user-images.githubusercontent.com/24326525/201564165-a998122c-919b-4237-96a7-465e88065e70.png)

**Comment by heliping** [comment]
```
[help4@dev23 rhuax]$ sea-orm-cli generate entity -u mysql://root:sanysany@10.100.0.211:3306/huaxrobot -o entities
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Database(MySqlDatabaseError { code: Some("42S22"), number: 1054, message: "Unknown column 'generation_expression' in 'field list'" })', /home/help4/.cargo/registry/src/github.com-1ecc6299db9ec823/sea-schema-0.10.2/src/mysql/discovery/executor/real.rs:29:14
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by tyt2y3** [comment]
The `expression: Option<ColumnExpression>` column is not essential (and not used anyway). But SeaSchema was meant to be a faithful representation, so I tried to include that.
We might be able to fill in `None` if such column does not exist. I remember such "try" logic exists somewhere with the SeaSchema codebase.

**Comment by billy1624** [comment]
MySQL version based selection. This? I guess.
https://github.com/SeaQL/sea-schema/blob/6b7ab04c891cf4d48418c2bfda689ef06d9fd5e4/src/mysql/query/index.rs#L58-L66

**Comment by tyt2y3** [comment]
Exactly

**Comment by billy1624** [comment]
Hey everyone, this will be patched on the next sea-orm-cli release.
https://github.com/SeaQL/sea-schema/pull/86

**Comment by billy1624** [comment]
This bug has been patched on [sea-schema 0.10.3](https://github.com/SeaQL/sea-schema/releases/tag/0.10.3).
Please re-install `sea-orm-cli` by executing `cargo install sea-orm-cli --force` in order to generate entities on MySQL 5.6
